src/platform/qt/ShaderSelector.cpp

```diff
@@ -66,33 +89,48 @@ void ShaderSelector::selectShader() {
 #if !defined(USE_LIBZIP) && !defined(USE_MINIZIP)
 	QString name = GBAApp::app()->getOpenDirectoryName(this, tr("Load shader"), path.absolutePath());
 #else
-	QString name = GBAApp::app()->getOpenFileName(this, tr("Load shader"), "mGBA Shaders (*.shader)", path.absolutePath());
+	QString filters = QStringLiteral("%1 (*.shader manifest.ini)").arg(tr("mGBA Shaders")), 
```

Comment:
```suggestion
	QString filters = QStringLiteral("%1 (*.shader manifest.ini)").arg(tr("mGBA Shaders"));
```

Reply:
Now how did that happen? I compiled and ran this. O.o; Did I fat-finger a vim command while preparing the PR?